.devcontainer/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN apt-get install -y python3 python3-venv libaugeas0 \
     && /opt/certbot/bin/pip install certbot
 
 # Install Porter
-RUN export PORTER_HOME=/home/vscode/.porter && curl -L https://cdn.porter.sh/latest/install-linux.sh | bash && chown -R vscode /home/vscode/.porter/
+RUN export PORTER_HOME=/home/vscode/.porter && curl -L https://cdn.porter.sh/latest/install-linux.sh | bash && /home/vscode/.porter/porter mixin install docker  && chown -R vscode /home/vscode/.porter/
```

Comment:
Why are using the docker mixin? Will that not fail with docker in docker in docker... ;)

Reply:
Chatting with @jjcollinge who reckon is should work. Guess we need to test it somehow.

Reply:
I am not sure it is a problem. Just a question

Reply:
Worked this out. I've added option to use the azure driver, and amended the docker file to use a container with the cnab driver. Tested as far as I could in current setup. Will likely need some work down the line.